package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@manuscripts/body-editor",
   "description": "Prosemirror components for editing and viewing manuscripts",
-  "version": "1.7.26",
+  "version": "1.7.24-LEAN-3429-2",
```

Comment:
The version does not sound correct. should be  `1.7.26-`

Reply:
this is from zax it's ok

Reply:
Just make sure you have the latest changes released before sending it to QA.